module/support/SoccerSimulator/src/SoccerSimulator.cpp

```diff
@@ -66,16 +67,14 @@ namespace support {
     using utility::math::angle::normalizeAngle;
     using utility::math::angle::vectorToBearing;
     using utility::math::coordinates::cartesianToSpherical;
-    using utility::math::matrix::Transform2D;
+    using utility::math::transform::worldToLocal;
     using utility::motion::kinematics::calculateRobotToIMU;
-    using utility::nusight::drawArrow;
-    using utility::nusight::drawSphere;
     using utility::nusight::graph;
     using utility::support::Expression;
 
     double triangle_wave(double t, double period) {
-        auto a = period;  // / 2.0;
-        auto k = t / a;
+        double a = period;  // / 2.0;
```

Comment:
What does this comment mean?

Reply:
Haven't a clue, it was there originally. I guess we used to use a half period at some point?

Reply:
I suppose it's just as confusing as the rest of this module